aiven/templates/endpoint_user_config_schema.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by go generate; DO NOT EDIT.
-// This file was generated at Tue 14 Jul 10:56:39 CEST 2020
```

Comment:
These changes are not relevant to Kafka ACL regex modification

Reply:
Good point, removed